### PR TITLE
Product page added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
Product Page-1

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
The code looks well structured and has good use of variables, good to merge.

File: product/product.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    padding: 1.5rem;
+}
+.single-card-img{
+    width: 15rem;
+    height: 20rem;
+}
+.single-card-right{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (manojasarsa) [inline product/product.css]: You can make a class of it and use it whenever needed.

File: product/product.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+}
+.single-card-right .tag-msg,
+.other-info{
+    font-size: 0.8rem;
+}
+.other-info li{
+    display: flex;
+    gap: 0.8rem;
+    align-items: center;
+    flex-direction: column;
```
Comment (manojasarsa) [inline product/product.css]: Here, it can be used.

**Review by manojasarsa — APPROVED** [review]
Rest is good to merge!!

File: product/product.css (hunk 1 above)
Comment (vaibhav18matere) [reply product/product.css]: got it, made the changes! thanks for pointing out !!

**Pull request merged**